Components/Alert.css

```diff
@@ -37,17 +37,17 @@
     display: flex;
     flex-direction: column;
     gap: 15px;
-    border: 3px solid var(--light-gray);
+    border: 1px solid black;
```

Comment:
For being consistent try to use css variables for colors like it is seen in the rest of your code